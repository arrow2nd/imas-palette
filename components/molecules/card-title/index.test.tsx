import { render } from '@testing-library/react'
import CardTitle from './index'

describe('CardTitle', () => {
  test('見た目が変化していないか', () => {
    const { container } = render(
      <CardTitle name="小早川紗枝" nameSuppl="Sae Kobayakawa" hex="#D967A3" />
    )

    expect(container).toMatchSnapshot()
  })
})
