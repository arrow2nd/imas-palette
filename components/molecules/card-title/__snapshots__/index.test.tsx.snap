// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardTitle 見た目が変化していないか 1`] = `
<div>
  <div
    class="flex flex-row items-center"
  >
    <div
      class="flex-none w-16 h-16 rounded-xl border border-gray-200"
      style="background-color: rgb(217, 103, 163);"
    />
    <div
      class="flex-1 ml-6"
      data-testid="name-label"
    >
      <p
        class="text-lg leading-6"
      >
        小早川紗枝
      </p>
      <p
        class="text-sm tracking-wide"
      >
        Sae Kobayakawa
      </p>
    </div>
  </div>
</div>
`;
